internal/logger/logger_test.go

```diff
@@ -125,7 +127,7 @@ func (t *LoggerTest) TestTextFormatLogs_LogLevelOFF() {
 	}
 
 	// Assert that nothing is logged when log level is OFF.
-	validateLogOutputAtSpecifiedFormatAndSeverity("json", config.OFF, expected)
+	validateLogOutputAtSpecifiedFormatAndSeverity(t.T(), "json", config.OFF, expected)
```

Comment:
Shouldn't we call assertion directly instead of function call?

ref: https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests

Reply:
This PR is to migrate from using jacobsa test libraries for the given packages. Yes, we will be working towards going DAMP in unit tests in future.